app/build.gradle

```diff
@@ -42,7 +42,6 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'androidx.constraintlayout:constraintlayout-solver:1.1.2'
     implementation 'androidx.cardview:cardview:1.0.0-beta01'
-    implementation 'androidx.constraintlayout:constraintlayout:1.1.2'
     implementation "com.android.support:appcompat-v7:${supportVer}"
```

Comment:
Support (com.android.support) libraries are going to be soon removed the new changes are given here (https://developer.android.com/topic/libraries/support-library/refactor) androidx dependencies are new.
Refer to Dojma or events to see implementation.